mmv1/products/compute/NetworkAttachment.yaml

```diff
@@ -13,7 +13,7 @@
 
 --- !ruby/object:Api::Resource
 name: 'NetworkAttachment'
-min_version: beta
```

Comment:
if https://github.com/GoogleCloudPlatform/magic-modules/pull/10966 get merged, we can remove this line.